web-animations-2/Overview.bs

```diff
@@ -216,18 +216,19 @@ follows:
 1.  Set |previous progress| based in the first condition that applies:
     <div class="switch">
 
-    : If |previous current time| is unresolved:
+    :   If |previous current time| is unresolved:
```

Comment:
Indent this list one level so it's properly nested under its `div`; then you won't need the blank line to make it work properly. You can also swap out the `div` for a `dl`, so it'll collapse into the Markdown `dl` in the output rather than remaining as a wrapper.

(Can't do a suggestion for this, since it woudl involve editting unchanged lines.)

Reply:
For swapping to a dl, I'm not sure how to preserve the switch formatting. In the generated css, `div.switch > dl > dt` gives the custom markers but if i switch to just a dl then of course it doesn't have the wrapping div so the custom markers aren't used.

Reply:
Hm, the selector should just be keying off of the `switch` class, not the `div` tagname. I'll note, tho, that in the current draft it's not being displayed at all; it's possible that the custom marker is only in Bikeshed's stylesheet and not the W3C stylesheet. ^_^

Reply:
That's curious, on https://drafts.csswg.org/web-animations-2/ the styles for switch are specified on the dl element rather than the wrapping div. I guess there's something wrong with my local bikeshed?

Reply:
Ah indeed, it's on `dl.switch`. Well, my point still stands that you should swap to a wrapper `dl` rather than a wrapper `div` so this actually works the way you intend. ^_^

Reply:
After fixing all of them it seems like my local bikeshed is generating css for `dl.switch` now 🤷‍♂️